CICDTemplate/README.md

```diff
@@ -0,0 +1,38 @@
+# Setup
```

Comment:
should this README be inside of your `.bitrise` directory? I will have another README for github action too

Reply:
@sleepylee Yep 👍